Cargo.toml

```diff
@@ -67,6 +67,9 @@ sha2 = { version = "0.10", optional = true }
 rsa = { version = "0.6.0", optional = true }
 ed25519-dalek = { version = "1.0.1", optional = true }
 
+# email formats
+email_address = "0.2.1"
```

Comment:
Not that I expect it to make any noticeable difference but I'd import this without default features, as we don't need serde support